### ISSUE TITLE
Large gaps in world

### ISSUE THREAD

**Issue by ghost** [webbukkit/dynmap#1795]
I already filled in the chunks in a 10x10 km area with WorldGuard, but after running DynMap to render the Surface map for an afternoon (still not finished), there's a weird gap in the render. See: 69.107.81.143:8123

**Comment by Cpt-Beefheart** [comment]
Sorry, your map isn't coming up.  With larger renders like that though, I've had gaps due to restarts before.

**Comment by gigapouch** [comment]
Did you use the "Surface" option for "world"?

**Comment by Cpt-Beefheart** [comment]
I still can't bring up your map to see what is going on.

**Comment by gigapouch** [comment]
Try looking at this screenshot:
http://69.107.81.143/MCServer/uploads/map.PNG

**Comment by Cpt-Beefheart** [comment]
Interesting!  I had similar with a large full render and the gaps seemed to coincide with server restarts.  My gaps were only a tile or few wide though, while yours seem to be a handful of chunks.  
Thoughts:
a) Do you have some weird settings on what parts of maps are visible and hidden?
B) did you explore your world prior or use a plugin to generate it all?
C) if you go in an empty area and do a small 100 block radius render does it then populate the missing chunks?
D) if C is correct, it's annoying, but is what I did.. Turn on the dynmap option to map on player moves, and fly along the strips. Ideally you won't have players on at this time.  
Regards,

**Comment by gigapouch** [comment]
I did explore a little when I first started the server. But I then filled in the rest of the chunks in the world with WorldGuard so it would be exactly 10x10km. When I teleport above the blank areas on the map, everything looks normal in Minecraft. What I will try for now is let dynmap run for several hours and see if it fills in those empty chunks. What I noticed too was that the map updates frequently to show what the players are building (awesome!). I will just fly along the gaps and see if dynmap thinks I am building something. If all else fails, I'll try C.

**Comment by Cpt-Beefheart** [comment]
Probably an issue with filling in the rest of the chunks did not work.  If that's the case then actually flying will fix it without changing the triggers since those chunks have probably never been generated.

**Comment by gigapouch** [comment]
They have been generated, because when I teleport to that location in Minecraft, it looks perfectly fine.

**Comment by Cpt-Beefheart** [comment]
Haha.. Well yes, the instant you go there they are generated, that's how Minecraft works.. You can not observe what we are talking about in game.  

If you want to see for yourself, you'll have to start a small radius render from the console centred on a black patch.  I am assuming the same gaps are in your flat map, and caves if you are doing caves.  The bugs with generated a whole world with a plugin are documented.

**Comment by gigapouch** [comment]
I'll try that once the fullrender of the "Surface" map is finished (I've been running it for about 6 hours a day for over a week and it still isn't finished).

**Comment by Cpt-Beefheart** [comment]
I think mine of similar size was a week and a half 24/7..

**Comment by gigapouch** [comment]
My computer would probably cost a lot of money if I left it on overnight. Besides, I don't want some humming machine near my bed. 


     On Friday, June 19, 2015 9:10 PM, Cpt-Beefheart <notifications@github.com> wrote:
   

 I think mine of similar size was a week and a half 24/7..

**Comment by Psy-Virus** [comment]
If this is still an issue with latest versions of Dynmap, please consider opening a new issue with updated details.
